Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -757,10 +757,10 @@ final class BuildOperationBuildSystemDelegateHandler: LLBuildBuildSystemDelegate
     }
 
     /// Invoked when an action taken before building emits output.
-    func preparationStepHadOutput(_ name: String, output: String) {
+    func preparationStepHadOutput(_ name: String, output: String, verboseOnly: Bool) {
```

Comment:
The purpose of this new parameter is a bit unclear at a glance from its name. Maybe it could be renamed or the doc comment on this function expanded to clarify this?